### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by steampunko** [daipenger/Sound-Physics#24]
Sorry, it's just been a bit since I've seen it updated

**Comment by KKZiomek** [comment]
Exactly. And are we at least going to get a 1.7.10 branch? At least half of people who play on mods still sit on servers that are version 1.7. I'm an owner of a major multiplayer server like this, and we would really like to enhance our gameplay with sound physics.

**Comment by Tsoccerguy3** [comment]
The project is Open Source , how could it be dead . ? You have a Git-hub account , If you wish enhancements to SP ,  then push the updated code for review .

**Comment by steampunko** [comment]
You know what I mean.. Plus, I can't code anything for the life of me

**Comment by Tsoccerguy3** [comment]
Dynamic Surroundings + Sound-Physics = Ear Candy , Oh wait a minute , candy doesn't go in your ear . lol. best not say that 3 times fast .
It's working for me and waiting till 1.13 final is out before I revisit this code base .
In the mean time , here are free online tutorials with lessons for you to do over the summer. 
https://www.quora.com/What-are-the-best-Java-tutorials
https://hackr.io/tutorials/learn-java

**Comment by SmokePuddingEveryday** [comment]
I believe the full 1.13 is out now. Will there be an update to the mod anytime soon?

**Comment by daipenger** [comment]
1.13 has been out for a while, but this is a forge mod. I don't know the current progress of forge, but I suspect it will still take 2-3 months for the first development builds of it. 1.13 has been a huge technical update.
However, my free time has become a lot less in the last two to three years and therefore I just don't want to wrap my head around fixing, improving (which this mod still needs to quite some extent) and updating it anymore.

But have I really been the only one on this planet earth to care enough to contribute to this mod in the past one and a half years since Cody released the source code of the mod? Yes. Apparently yes. Well, I am not even using it anymore myself. I also just lost interest and forgot about it and I think, so did other potential developers. I am just leaving this fork for another lonely developer who might still rediscover it, but I do not think that this will ever happen anymore.
I am not earning money from my mods, neither that nor my other mods, my only motive was that I was using it myself and wanted it updated. Both is not true anymore.

**Comment by Ynaught** [comment]
> 
> 
> Dynamic Surroundings + Sound-Physics = Ear Candy , Oh wait a minute , candy doesn't go in your ear . lol. best not say that 3 times fast .
> It's working for me and waiting till 1.13 final is out before I revisit this code base .
> In the mean time , here are free online tutorials with lessons for you to do over the summer.
> https://www.quora.com/What-are-the-best-Java-tutorials
> https://hackr.io/tutorials/learn-java

1.14.4 and 1.15.2 are the 2 branches everyone seems to be making mods for, never seen a mod for 1.13
